src/OctostacheCmd/OctostacheCmd.csproj

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="octostache" Version="2.13.0" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.21308.1" />
```

Comment:
Is there an advantage to going with the beta version?

Reply:
This is going to sound wrong, but I don't think there is a non-beta/alpha version despite being 2.0.0. At least I can't find one on [nuget.org](https://www.nuget.org/packages/System.CommandLine) 🤷‍♂️

Reply:
I've used [this one](https://www.nuget.org/packages/McMaster.Extensions.CommandLineUtils) in the past.  It's a little more object oriented in the way it handles arguments.  If you like `System.CommandLine` better, I'm good with that, too.

Reply:
It just seemed super simple, I'm not really tied to it tbh. I mostly wanted to try it out because I hadn't heard of it before.